public/locales/en/modules/calendar.json

```diff
@@ -7,6 +7,9 @@
       "useSonarrv4": {
         "label": "Use Sonarr v4 API"
       },
+      "useRadarrv5": {
```

Comment:
Instead of adding a new option; how about making a select dropdown where you can select the API version?

Reply:
I mean to be fair there’s only V5 or V4 and we already have a toggle for sonarr V4 and V5

Reply:
But how about we detect the version via an API call ? It would be nice to have that as the result of the “test integration” button on the future integrations tab in the config.

Reply:
This was a quick fix to include in the new version since we get a lot of reports for it.
This would be a nice new feature, but that's the fix for now that follows the same way it was done for sonarr.